applications/DEMApplication/python_scripts/DEM_analysis_stage.py

```diff
@@ -653,6 +663,13 @@ def PrintResultsForGid(self, time):
             if self.DEM_parameters["post_vtk_option"].GetBool():
                 self.vtk_output.WriteResults(self.time)
 
+        self.file_msh = self.demio.GetMultiFileListName(self.problem_name + "_" + "%.12g"%time + ".post.msh")
+
+        self.PrintResultsForGidAdditionalOperations()
```

Comment:
If it's at the end of a method, there's no need to add an AdditionalOperations. You can just override the method, call overriden and add the additional lines after that.